pyproject.toml

```diff
@@ -3,7 +3,7 @@ build-backend = "mesonpy"
 requires = [
     "meson >= 1.2.0",
     "meson-python >= 0.13.1",
-    "pybind11 >= 2.12.0",
+    "pybind11 >= 2.13.0",
```

Comment:
2.13.1 was released the day after 2.13.0 ((IIRC with a free-threading related fix as well), so I'd always use .1 as the minimum version